.gitignore

```diff
@@ -22,5 +22,7 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-dist/
+public/
 
+#AWS KEYS
+server/config.js
```

Comment:
A new line at the EOF is important to many linters. The little red symbols there are requesting it.

Reply:
What is EOF?

Reply:
End of file